src/Execute/RealDeviceInstance.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {DeviceSpec} from '../Backend/Spec';
+
+// What is `RealDeviceInstance` class?
+// Suppose a case that some tizen tvs are. dtv0, dtv1, ... dtvn_1
+// the dtv's spec can be just only one
+// However, each dtv should be handled one by one
+// At the time, this class expresses an instance for a real device
+class RealDeviceInstance {
```

Comment:
I am not sure the meaning of `Real` and `Insatance` here. Can we just call this `Device`?
Thinking about `Real`, will there be, e.g., `VirtualDeviceInstance` later? I wonder what other `DeviceInstance` will be in future. Could you share some example?
Similarly, thinking about `Instance`, will there be, e.g., `RealDeviceClass` later?

Reply:
It makes sense. However, `Device` can be ambiguous. The situation can be explained as comments.

```
// Suppose a case that some tizen tvs are. dtv0, dtv1, ... dtvn_1
// the dtv's spec can be just only one
// However, each dtv should be handled one by one
// At the time, this class expresses an instance for a real device
```

I can agree on excepting `Real` but I think `Instance` is necessary.

Reply:
I also agree about `Real` part. I think `Real` means nothing about this. I will remove that expression. 
but about `Instance` I think remaining this could be more nice. I will change to `DeviceInstance` would look more nice.

Reply:
> Similarly, thinking about `Instance`, will there be, e.g., `RealDeviceClass` later?

No, `DeviceClass` will not implemented. I think `DeviceInstance` could classified by `DeviceSpec`.

Reply:
I am still wonder why `Instance` is needed in naming. So, to me it's kinda hard to understand what this class is for... Could you explain why it is _necessary_ and why it makes code _more nice_? 

IMHO, 
When a class is instantiated,  for example, `let a = new Foo()`.
everyone know `a` is an instance of `Foo`. So we don't normally use `FooInstance` as class name. 

By naming `FooInstance` (not `Foo`),
for me, I assume that an instance of class `Foo` is mapped with one instance of `FooInstance`...
It's kinda complex to understand.
Instead of creating `FooInstance`, can we move methods and vars of `FooInstance` into `Foo` and remove `FooInstance`?

I guess maybe I am missing something here... More explanation would help me to understand why tthis class is needed.

Reply:
Okay.
agree on `instance`: +1 (@YongseopKim)
disagree on `instance`: +1 (@hyunsik-yoon)

Now it depends on you, @struss. If you disagree on `instance` and others would be okay, I'll be okay too.

Reply:
um... I use `Instance` to emphasize A single Device with `DeviceSpec`. but follow from @hyunsik-yoon opinion, make class name with `Instance` will make redundant meaning. 

so, I will add one point to @hyunsik-yoon 's opinion.